modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/odata/ExpressionVisitorImpl.java

```diff
@@ -309,6 +328,23 @@ private JSONObject _getConjunctionJSONObject(
 		);
 	}
 
+	private Object _getDuration(String literal)
```

Comment:
This method should return a java.time.Duration, right?

Reply:
Right